test/e2e/framework/pod/utils.go

```diff
@@ -294,18 +297,20 @@ func FindContainerStatusInPod(pod *v1.Pod, containerName string) *v1.ContainerSt
 // VerifyCgroupValue verifies that the given cgroup path has the expected value in
 // the specified container of the pod. It execs into the container to retrive the
 // cgroup value and compares it against the expected value.
-func VerifyCgroupValue(f *framework.Framework, pod *v1.Pod, cName, cgPath, expectedCgValue string) error {
+func VerifyCgroupValue(f *framework.Framework, pod *v1.Pod, cName, cgPath string, expectedCgValues ...string) error {
```

Comment:
Function comment needs to be updated. It would be reasonable to assume that *all* values must be set, but what is implemented seems to be *at least one* value must be set.

Reply:
updated!